ecommerce/src/main/java/app/config/WebSecurityConfig.java

```diff
@@ -26,7 +26,8 @@ public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception
 
 	@Override
 	protected void configure(HttpSecurity http) throws Exception {
-		http.csrf().disable().authorizeRequests().antMatchers("/", "/login", "/logout", "/assets/**","/products/**","/blogs/**").permitAll()
+		http.csrf().disable().authorizeRequests().antMatchers("/", "/login", "/logout", "/assets/**",
+				"/products/**","/blogs/**","/cart").permitAll()
```

Comment:
```suggestion
		http.csrf().disable().authorizeRequests().antMatchers("/**").permitAll()
```
ri dc ko bạn, chứ ri riết thêm nhìu controller cái ngất ngang